test/syscalls/linux/proc_defaults.cc

```diff
@@ -0,0 +1,89 @@
+// Copyright 2021 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <linux/sem.h>
+#include <linux/shm.h>
+
+#include "absl/strings/numbers.h"
+#include "absl/strings/str_split.h"
+#include "gtest/gtest.h"
+#include "test/util/fs_util.h"
+#include "test/util/test_util.h"
+
+namespace gvisor {
+namespace testing {
+namespace {
+
+std::vector<std::string> saved_argv;  // NOLINT
```

Comment:
Why does this need to be saved in a global variable? This doesn't seem to be used anywhere. Also, why is this NOLINT?

Reply:
This was (hastily) copied, along with the main function from proc tests. Since it's not useful in these tests, I removed it, along with `main`, and added `"//test/util:test_main"` to test dependencies.